File: Java/219.java
import java.util.Scanner;

/**
 *
 * 219 - La Lotería de la Peña Atlética - Iván - https://github.com/wildfireOfMine
 */
public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int casos = sc.nextInt();
        for (int i = 1; i<=casos; i++) {
            int numeros = sc.nextInt();
            int puntuaje = 0;
            for (int j = 1; j<=numeros; j++) {
                int numerito = sc.nextInt();
                if (numerito % 2 == 0) {
                    puntuaje++;
                }
            }
            System.out.println(puntuaje);
        }
    }

}

File: C++/219.cpp
#include <iostream>

using namespace std;

// 219 - La Lotería de la Peña Atlética - Iván - https://github.com/wildfireOfMine

int main()
{
    int loop = 0;
    cin >> loop;
    for(int i = 0; i<loop; i++) {
        int counter = 0;
        int subloop = 0;
        cin >> subloop;
        for (int j = 0; j<subloop; j++) {
            int number = 0;
            cin >> number;
            if (number % 2 == 0){
                counter++;
            }
        }
        cout << counter << endl;
    }

}
